artemis-core-client/src/main/java/org/apache/activemq/artemis/core/protocol/core/impl/wireformat/SessionQueueQueryResponseMessage_V3.java

```diff
@@ -475,6 +501,11 @@ public boolean equals(Object obj) {
             return false;
       } else if (!ringSize.equals(other.ringSize))
          return false;
+      if (enabled == enabled) {
```

Comment:
This is wrong I think

Reply:
good spot, https://github.com/apache/activemq-artemis/pull/3174 fix here.